networkx/algorithms/chordal/chordal_alg.py

```diff
@@ -242,6 +243,48 @@ def chordal_graph_treewidth(G):
         max_clique = max(max_clique,len(clique))
     return max_clique - 1
 
+def chordal_simplicial_vertex(G):
```

Comment:
Use the decorator `@not_implemnted_for('directed','multigraph')` here. This will raise `NetworkXNotImplemented`, which seems much more relevant.